src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -4546,6 +4546,132 @@ default Emote getEmoteById(long id)
     @CheckReturnValue
     AuditableRestAction<Void> unban(@Nonnull String userId);
 
+    /**
+     * Puts the specified Member in time out in this {@link net.dv8tion.jda.api.entities.Guild Guild} until the specified date.
+     * <br>While a Member is in time out, they cannot reply, send messages, react and speak in voice channels.
+     *
+     * <p>Possible {@link net.dv8tion.jda.api.requests.ErrorResponse ErrorResponses} caused by
+     * the returned {@link net.dv8tion.jda.api.requests.RestAction RestAction} include the following:
+     * <ul>
+     *     <li>{@link net.dv8tion.jda.api.requests.ErrorResponse#MISSING_PERMISSIONS MISSING_PERMISSIONS}
+     *     <br>The target Member cannot be kicked due to a permission discrepancy</li>
```

Comment:
```suggestion
     *     <br>The target Member cannot be timeouted due to a permission discrepancy</li>
```

Reply:
timed out?

Reply:
> timed out?
Sounds better yes

Reply:
"cannot be placed into time-out due to..."